packages/@aws-cdk/aws-stepfunctions-tasks/lib/sagemaker/create-transform-job.ts

```diff
@@ -173,6 +181,15 @@ export class SageMakerCreateTransformJob extends sfn.TaskStateBase {
     };
   }
 
+  private renderModelClientConfig(config: ModelClientConfig): { [key: string]: any } {
+    return {
+      ModelClientConfig: {
+        InvocationsMaxRetries: config?.invocationsMaxRetries,
```

Comment:
is the `config?` required here - it's a required property of this method

Reply:
Nice catch. I was hoping this would be enough but ended up using it as a required arg. Should've fixed it before requesting review, my bad.

Reply:
> represented as tokens (i.e. if they're passed in as parameters to the stack)

I don't think I've used this style of passing params to stacks but am curious and will read up more on how this is done :). I think I did it correctly, but please let me know if I didn't. Thanks!